code/modules/mob/mob_defines.dm

```diff
@@ -229,3 +229,8 @@
 	var/active_thinking_indicator
 	/// User is thinking in character. Used to revert to thinking state after stop_typing
 	var/thinking_IC = FALSE
+
+	///Who is gunpointing at our person?
+	var/list/datum/component/gunpoint/gunpointed = list()
+	///The effect from gunpointing, to make sure we dont create multiples
+	var/obj/effect/overlay/gunpoint_effect/gp_effect
```

Comment:
There's no chance at all that making this component coupled to living is a good idea at all. I can't see a world where this is accepted. What are you trying to do with this?

Reply:
I need a way to find out if the person is under any gunpoint. Right now, the component is created on the shooter. If there are multiple gunpoints on the target, I need to somehow check if removing one of the gunpoints should remove the visual effect.